packages/sc-project-initializer/commands/init/README.md

```diff
@@ -43,33 +30,30 @@ Prerequisites :
 
 These keys will be the ones used by the deployer script to interact with the blockchain.
 
-The following command will build your contract and create the deployer associated:
+The following command will build your contract and create the deployer associated. Then it will be deployed. 
 It assumes your contract entrypoint is `assembly/main.ts`
 
-```shell
-npm run build
-```
-
-Then deploy your contract with:
-
 ```shell
 npm run deploy
 ```
 
 This command will deploy your smart contract on Massa's network corresponding to the given node.
 
-When you deploy a contract the function `constructor` is called. If you want to deploy more contracts or pass parameter to the constructor function you have to modify `src/deploy.ts`. Even if you want the default behavior it's cool to take a look at it.
+When the deployment transaction is executed onchain, the `constructor` function of your smart contract `assembly/main.ts` will be called.
```

Comment:
When the deployment operation is executed on-chain, the `constructor` function of your smart contract `assembly/main.ts` will be called.